tests/Oryx.RuntimeImage.Tests/NodeImagesTest.cs

```diff
@@ -98,6 +98,30 @@ public void NodeVersionMatchesImageName(string nodeTag, string nodeVersion)
                 result.GetDebugInfo());
         }
 
+        [Theory]
+        [InlineData("10.1", "5.6.0")]
```

Comment:
This variation to verify that we only upgrade the major version `6`

Reply:
can you write this comments in the test itself?